sdk/azcore/response.go

```diff
@@ -114,7 +114,7 @@ func (r *Response) UnmarshalAsXML(v interface{}) error {
 // Drain reads the response body to completion then closes it.  The bytes read are discarded.
 func (r *Response) Drain() {
 	if r != nil && r.Body != nil {
-		io.Copy(ioutil.Discard, r.Body)
+		_, _ = io.Copy(ioutil.Discard, r.Body)
```

Comment:
We dont care about the error here?

Reply:
Correct, we're just draining the response body.